Replace nginx with istio (#4205)
kyma-project/kyma: resources/application-connector/charts/connector-service/README.md

- **tokenLength** - Length of registration tokens. The default value is `64`.
- **appTokenExpirationMinutes** - Time after which tokens for Applications expire and are no longer valid. The default value is `5` minutes.
- **runtimeTokenExpirationMinutes** - Time after which tokens for Runtimes expire and are no longer valid. The default value is `10` minutes.

> - **caSecretName** - Namespace and Name of the secret, in Namespace/Name format, which contains the certificate and key used for signing client certificates. The default value is `kyma-integration/application-connector-ca-certs`.

Reviewer: ```suggestion
- **caSecretName** - Namespace and the name of the Secret which stores the certificate and key used to sign client certificates. Requires the `Namespace/secret name` format. The default value is `kyma-integration/application-connector-ca-certs`.
```

> - **rootCACertificateSecretName** - Namespace and Name of the secret, in Namespace/Name format, which contains root CA (Certificate Authority) Certificate in case certificates are singed by intermediate CA.

Reviewer: ```suggestion
- **rootCACertificateSecretName** - Namespace and the name of the Secret which stores the root CA (Certificate Authority) Certificate. Requires the `Namespace/secret name` format.
```

Reviewer: There's no default value for this one, is this by design?

Author: The default is empty, as this parameter is not required.
